lib/repository.js

```diff
@@ -653,7 +653,11 @@ Repository.prototype.fetch = function(
         return remote.disconnect();
       });
     })
-    .then(finallyFn, finallyFn);
+    .then(finallyFn)
```

Comment:
We could simplify this further by using `.finally` in [then/promise](https://github.com/then/promise/blob/master/src/finally.js#L6-L16)